src/bson.rs

```diff
@@ -408,6 +405,11 @@ impl Bson {
             Bson::Symbol(v) => json!({ "$symbol": v }),
             #[cfg(feature = "decimal128")]
             Bson::Decimal128(ref v) => json!({ "$numberDecimal": v.to_string() }),
+            #[cfg(not(feature = "decimal128"))]
+            Bson::Decimal128(_) => panic!(
```

Comment:
this is a bit tricky since we cannot return an error here, but we also cannot come up with a reasonable value to return. I opted to panic, but we could also fail more silently here by returning an incomplete value (e.g. `{ "$numberDecimal": null }` or just `null`).

Reply:
I added a note in the docstring for this method to call it out so users won't be too surprised if they run into it.